### PR TITLE
Include PR in merge title+messgae

### PR DESCRIPTION
**Pull request by lpil**
### Checklist

* [x] I have read the [Contributor Guide](../../CONTRIBUTING.md)
* [x] I have read and agree to the [Code of Conduct](../../CODE_OF_CONDUCT.md)
* [x] I have added a description of my changes and why I'd like them included in the section below

### Description of Changes

It does what it says ~on the tin~ in the title

### CODE REVIEW

File: src/merge.rs [hunk 1]
```diff
@@ -69,19 +69,19 @@ pub async fn queue(
                     // statuses we actually cared about have all passed, so we "should"
                     // be ok
                     MergeableState::Clean | MergeableState::HasHooks | MergeableState::Unstable => {
-                        let mut merge = prh
+                        let merge = prh
                             .merge(pr_number)
-                            .title(pr.title)
+                            .title(format!("{} (#{})", pr.title, pr_number))
```
Comment (Jake-Shadle) [inline src/merge.rs:74]: Does this actually create a hyperlink to the pr?

File: src/merge.rs [hunk 2]
```diff
@@ -69,19 +69,19 @@ pub async fn queue(
...
                                     .merge_method
                                     .unwrap_or(octocrab::params::pulls::MergeMethod::Merge),
-                            );
-
-                        if let Some(body) = pr.body {
-                            merge = merge.message(body);
-                        }
+                            )
+                            .message(match pr.body {
+                                Some(body) => format!("{}\n\n{}", body, pr.url),
```
Comment (Jake-Shadle) [inline src/merge.rs:82]: Is this needed? Won't it just duplicate the pr url that should be available in the commit title where everyone is used to it?

**Review by Jake-Shadle — CHANGES_REQUESTED** [review]

File: src/merge.rs (hunk 1 above)
Comment (repi) [reply src/merge.rs:74]: Think GitHub automatically creates a link for it, just as it does in with the commit messages when they reference issues/PRs. 

But good to verify!

Comment (lpil) [reply src/merge.rs:74]: It does, I checked this commit https://github.com/EmbarkStudios/octobors/commit/9c0e83735980d2e7dcc0338cebbfc75ee5ed44ba

File: src/merge.rs (hunk 2 above)
Comment (lpil) [reply src/merge.rs:82]: The commit title is only clickable in the GitHub UI (where the link is largely hidden), I added this link so you can still click it in the terminal and in your editor, etc.

Comment (Jake-Shadle) [reply src/merge.rs:82]: Ok, I guess we can try it out.

**Review by Jake-Shadle — APPROVED** [review]

**Comment by lpil** [comment]
Thank you

**Pull request merged**